dmutils/filters.py

```diff
@@ -96,3 +98,24 @@ def preserve_line_breaks(eval_ctx, value):
         result = Markup(result)
 
     return result
+
+
+def sub_country_codes(text):
+    """Replace country codes with country common name
+
+    :param text:  text containing country code in the form 'country:TLD'
+    :return:  text containg country names, or TLDs if not found
```

Comment:
A small nitpick but it's not the TLD - it's the ISO alpha-2 code (see https://www.gov.uk/government/publications/iso-country-codes--2).

Reply:
```suggestion
        :param text:    text containing country codes in the form 'country:AA'
                    where AA is a two-letter ISO 3166-2 alpha-2 country code
        :return:        text containing country names, or the country code if not found
```